lib/rabbit_feed/configuration.rb

```diff
@@ -42,6 +43,7 @@ def connection_options
         options[:heartbeat] = heartbeat if heartbeat
         options[:connect_timeout] = connect_timeout if connect_timeout
         options[:host] = host if host
+        options[:hosts] = hosts if hosts
```

Comment:
@calo81 

>  is this property being picked up automatically by Bunny?

Yes

> I mean where is this value actually used?

`connection_options` are what is passed to Bunny, see: https://github.com/simplybusiness/rabbit_feed/blob/master/lib/rabbit_feed/connection_concern.rb#L53

Bunny is what is actually uses the configs. RabbitFeed allows you to specify configs in yaml format and passes them to Bunny for you.